main/src/test/java/org/mobilitydata/gtfsvalidator/validator/RouteAgencyIdValidatorTest.java

```diff
@@ -0,0 +1,71 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.MissingRecommendedFieldNotice;
+import org.mobilitydata.gtfsvalidator.notice.MissingRequiredFieldNotice;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.table.*;
+
+public class RouteAgencyIdValidatorTest {
+
+  public static GtfsRoute createRoute(
```

Comment:
Per advice in previous PR, I'd inline these helper functions and drop calls to setBlah(null) as appropriate.

Reply:
No problem doing this at all, just FYI this is a convention used in a many test fixtures, so I was copying those. Not sure if it's worth addressing elsewhere...